test/stresses.jl

```diff
@@ -29,6 +29,62 @@ include("testcases.jl")
     end
 
 
+    a = 10.0  # slightly compressed and twisted
```

Comment:
Again feels like a lot of repeated code for just changing a value. Just make a function and call it twice or so.